HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw

```diff
@@ -1,61 +1,153 @@
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="VmCredentialRequest.CancelText" xml:space="preserve">
+    <value>Cancel</value>
+  </data>
+  <data name="VmCredentialRequest.Description1" xml:space="preserve">
+    <value>DevHome needs VM administrator credentials to install DevSetupAgent service.</value>
+  </data>
+  <data name="VmCredentialRequest.Description2" xml:space="preserve">
+    <value>Please enter credential for the VM administrator account.</value>
+  </data>
+  <data name="VmCredentialRequest.OkText" xml:space="preserve">
+    <value>Ok</value>
+  </data>
+  <data name="VmCredentialRequest.PasswordErrorMsg" xml:space="preserve">
+    <value>Password is required</value>
+  </data>
+  <data name="VmCredentialRequest.PasswordLabel" xml:space="preserve">
+    <value>Password</value>
+  </data>
+  <data name="VmCredentialRequest.Title" xml:space="preserve">
+    <value>Hyper-V DevHome Extension credential request</value>
+  </data>
+  <data name="VmCredentialRequest.UsernameErrorMsg" xml:space="preserve">
+    <value>Username is required</value>
+  </data>
+  <data name="VmCredentialRequest.UsernameLabel" xml:space="preserve">
+    <value>Admin username</value>
+  </data>
+  <data name="WaitForLoginRequest.Description" xml:space="preserve">
+    <value>Please login to &lt;Name&gt; Hyper-V VM to continue configuration. Click Ok after logging on.</value>
+  </data>
+  <data name="WaitForLoginRequest.Title" xml:space="preserve">
+    <value>Hyper-V DevHome Extension login request.</value>

```

Comment:
Dev Home, two words
```suggestion
    <value>Hyper-V Dev Home Extension login request</value>
```
The convention set by the other extensions would make this "Dev Home Hyper-V Extension". Is this named differently?